run_fvs.sh

```diff
@@ -12,13 +12,23 @@ git status
 docker exec -t host1 bash -c 'docker rm -f $(docker ps -qa) ; \
                               docker rmi $(docker images -qa)' || true
 docker rm -f host1 || true
+
+docker pull busybox:latest
```

Comment:
It would be great if this new block of code doesn't incur extra run time if it doesn't need to.

The docker pull is pretty safe - it won't do anything (significant) if the image is already pulled
However, the save is a bit wasteful. It would be very easy to just have a simple if at the front to make it not do anything if the file exists already.
`if [ ! -e busybox.tar ] ; then docker save --output busybox.tar busybox:latest ; fi